index.html

```diff
@@ -703,15 +702,25 @@
           context</a>.
         </p>
         <p>
-          The <dfn>set of presentations</dfn>, initially empty, contains the
-          <a>presentations</a> created by the <a>controlling browsing
-          contexts</a> for the controlling user agent (or a specific user
-          profile within that user agent). The <a>set of presentations</a> is
-          represented by a list of tuples, where each tuple consists of a
-          <a>presentation URL</a>, a <a>presentation identifier</a>, and the
-          <a>presentation connection</a> itself. The <a>presentation URL</a>
-          and <a>presentation identifier</a> uniquely identify the
-          <a>presentation</a>.
+          The <dfn>set of controlling presentations</dfn>, initially empty,
+          contains the <a>presentation connections</a> created by the
+          <a>controlling browsing contexts</a> for the <a>controlling user
+          agent</a> (or a specific user profile within that user agent). The
+          <a>set of controlling presentations</a> is represented by a list of
+          <a>PresentationConnection</a> objects that represent the underlying
+          <a>presentation connections</a>. The <a>presentation URL</a> and
+          <a>presentation identifier</a> uniquely identify a <a>presentation
+          connection</a> in this set.
```

Comment:
This is not actually true, I think. Page A can start a presentation with (url, id) and page B can call new PresentationRequest(url).connect(id) to create a second PresentationConnection with the same (url,id).

So the set of controlling presentations may allow duplicates.